easybuild/easyconfigs/m/matplotlib/matplotlib-1.4.3-intel-2015a-Python-3.4.3.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = "PythonPackage"
+
+name = 'matplotlib'
+version = '1.4.3'
+
+homepage = 'http://matplotlib.org'
+description = """matplotlib is a python 2D plotting library which produces publication quality figures in a variety of
+ hardcopy formats and interactive environments across platforms. matplotlib can be used in python scripts, the python
+ and ipython shell, web application servers, and six graphical user interface toolkits."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+patches = [('matplotlib-%(version)s_Qhull-intel-fix.patch')]
+
+python = "Python"
+pythonversion = '3.4.3'
+pyshortver = ".".join(pythonversion.split(".")[:-1])
+
+versionsuffix = "-%s-%s" % (python, pythonversion)
+
+dependencies = [
+    (python, pythonversion),
+    ('freetype', '2.5.5'),
+    ('libpng', '1.6.16'),
+]
+
+# is this actually needed? the PythonPackage will check if import matplotlib succeeds, which is as good as this?
+pyprefix = 'lib/python%s/site-packages' % pyshortver
```

Comment:
this is the sanity check, that's different from the 'test' right?
Both check something different, doesn't hurt to have a sanity check, and a test.